Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+** Generally its a team name and one or more names. Sometimes, it's a third party (SaaS).
+** You never want to leave the service owner blank, and you always want a Mozilla contact as service owner.
+** You always want at least a team name, as people change teams, etc.
+* Other fields will be automatically filled in or filled in at a later time.
+
+=== General notes (5-10 minutes) ===
+
+At this point you want to have a good understanding of the service:
+
+* Where is it running? Which platforms?
+* What is it running, what kind of software or techonologies?
+* How is it structured, architected? Do you have your data flow diagram?
+* Do you have some documentation links?
+* Any URL to access the service?
+* How is administration performed?
+* How is authorization (login) performed?
+* What is logged, where?
+
+Anything of interest may go in the RRA general notes. This can be used during the description  of the service as well if you hear interesting details that did not fit in the description.
+
+{| class="wikitable"
+|-
+! <span style="color:red;">WARNING</span>
+|-
+|
+At the end of this exercise, you want to be able to reformulate how the service works, with your own words, and have the
+RRA participants tell you if that understanding is correct. This step is crucial.
+|}
+
+=== Data Dictionary (5-10 minutes) ===
+
+We want an exhaustive list of data types. While some types may be sometimes missing as this is a best-effort type discussion, it is important to ensure we're missing as little as possible.
+
+* Ask the team to help you list which data is handled by the application, such as:
+** Specific configuration data (on disk or in RAM).
+** Credentials used by the applications (keys, logins, etc.).
+** Potentially program code/script (such as if the code is stored on a Git repository, downloaded and ran by the service).
+** Ask again and specify you want to make sure they've listed all sensitive data they can think of.
+* Set the data classification for each data type in the dictionary, such as "PUBLIC", "CONFIDENTIAL", etc. Mozilla uses standard classification levels.
+* If some compensating controls are mentioned or there are more details about the data you can fill them in as well.
+** For example "User attributes, classified INTERNAL, protected by LDAP authentication, used to find out the user
+t-shirt size".
+* Based on this dictionary/catalog of data, figure out which is the main data classification the service is handling. If unsure, ask the team.
+** For example "mock is mainly used to serve package files that are PUBLIC." (public main data classification).
+** Set this in the header section at the top of the RRA template.
+
+=== RRA Risk table (5-10 minutes) ===
+
+This is where the risk impact assessment is done, and where the probability for these impact is roughly estimated.
+The risk impact is using ''probable impact'', that is, worst-case scenario impacts that seem possible.
```

Comment:
This is the most import part of the RRA and yet it has less explanation than the rest of the document. I think there should be more details as to what the expectations are.